src/main/kotlin/build/buf/gradle/BufYamlGenerator.kt

```diff
@@ -29,16 +29,17 @@ internal class BufYamlGenerator {
         // Emit a module for each discovered workspace, copying ignores and concatenating their
         // paths with the module root.
         val modules =
-            protoDirs.map { dir ->
-                if (ignores.isEmpty()) {
-                    mapOf("path" to dir)
-                } else {
-                    mapOf(
-                        "path" to dir,
-                        "breaking" to mapOf("ignore" to ignores.map { "$dir/$it" }),
-                    )
+            protoDirs
+                .map { dir ->
+                    if (ignores.isEmpty()) {
+                        mapOf("path" to dir)
+                    } else {
+                        mapOf(
+                            "path" to dir,
+                            "breaking" to mapOf("ignore" to ignores.map { "$dir/$it" }),
+                        )
+                    }
```

Comment:
This is just a format change right? Was this the output of running something like `gradlew spotlessApply`? (If so, maybe we should verify formatting in CI checks -- not an action for this PR, just a comment about a possible future improvement.)

Reply:
It was a manual change but I can revert to make the PR cleaner

Reply:
Formatting is verified in CI.

Reply:
Reverted